app/views/spree/admin/vendor_settings/edit.html.erb

```diff
@@ -32,6 +32,12 @@
         <%= f.text_area :contact_us, class: 'form-control' %>
       <% end %>
     </div>
+    <div class="form-group">
+      <%= f.field_container :notification_email do %>
+        <%= f.label :notification_email %>
+        <%= f.email_field :notification_email, class: 'form-control' %>
```

Comment:
Please add `:notification_email` to `Spree::PermittedAttributes.vendor_attributes` here config/initializers/spree_permitted_attributes.rb